gflib/text.h

```diff
@@ -150,6 +150,8 @@ void DecompressGlyphTile(const void *src_, void *dest_);
 void CopyGlyphToWindow(struct TextPrinter *x);
 void ClearTextSpan(struct TextPrinter *textPrinter, u32 width);
 
+#define NUM_FRAMES_AUTO_SCROLL_DELAY 49
```

Comment:
any particular reason it is defined in the middle here? I would personally move it to the top in the enum section

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/5054/commits/5dd1129655dd8cbf02b2bc1a4747e8e1240cafcc